src/lib/flags.ts

```diff
@@ -109,6 +109,10 @@ const FEATURE_FLAGS = {
 
   CRAFTING: betaTimeBasedFeatureFlag(SEASONS["Better Together"].startDate),
 
+  TRASH_BIN_CARRY_OVER_LIMIT: timeBasedFeatureFlag(
+    new Date("2025-08-08T00:00:00Z"),
```

Comment:
Maybe you can make it an additional day so we can have more testing
```suggestion
    new Date("2025-08-09T00:00:00Z"),
```